diff --git a/airdcpp-core/CMakeLists.txt b/airdcpp-core/CMakeLists.txt
index 1493877..9c0ad0b 100644
--- a/airdcpp-core/CMakeLists.txt
+++ b/airdcpp-core/CMakeLists.txt
@@ -58,7 +58,7 @@ find_package (OpenSSL REQUIRED)
 find_package (miniupnpc REQUIRED)
 find_package (leveldb REQUIRED)
 find_package (maxminddb REQUIRED)
-find_package (Boost 1.67.0 COMPONENTS regex thread REQUIRED)
+find_package (Boost 1.67.0 COMPONENTS thread REQUIRED)
 
 if (NOT BUILD_SHARED_LIBS)
   # Shared LevelDB libraries are linked against Snappy already
@@ -211,7 +211,7 @@ target_link_libraries (${PROJECT_NAME}
 	maxminddb::maxminddb
 	
 	Boost::thread
-	Boost::regex
+	Boost::headers
 
 	${airdcpp_extra_libs} 
 )
